classes/fields/pick.php

```diff
@@ -1409,6 +1409,12 @@ public function build_dfv_field_item_data_recurse_item( $item_id, $item_title, $
 	 * {@inheritdoc}
 	 */
 	public function validate( $value, $name = null, $options = null, $fields = null, $pod = null, $id = null, $params = null ) {
+		$return = parent::validate( $value, $name, $options, $fields, $pod, $id, $params );
+
+		$errors = array();
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 286c7f4f9eacf95ee0cc277c46919620</sub>